client/src/libs/enums/user/user.js

```diff
@@ -0,0 +1 @@
+export { UserPayloadKey } from 'shared/src/common/enums/enums';
```

Comment:
Move it to packages/users/libs/enums, do the same one for another PayloadKey/ The main idea, if we decide to drop one namespace it should be only from packages/